src/java/org/sakaiproject/component/app/scheduler/jobs/ClearRTTMail.java

```diff
@@ -66,7 +61,7 @@ public void execute(JobExecutionContext arg0) throws JobExecutionException {
 				Instant d = messageHeader.getInstant();
 
 				//1 hours
-				Instant cal2 = Instant.now().minus(1, ChronoUnit.HOURS);
+				Instant cal2 = Instant.now().plus(1, ChronoUnit.HOURS);
```

Comment:
Not sure why this should change from minus to plus? Logic here seems to be to delete messages that are more than 1 hour old, so before now less 1 hour.

Reply:
Instant: 2019-11-12T15:11:40.284Z
plus: 2019-11-12T16:11:40.284Z
minus: 2019-11-12T14:11:40.284Z
Calendar: Tue Nov 12 16:11:40 SAST 2019

`public static void main(String[] args) {
		// TODO Auto-generated method stub
	    Instant timestamp = Instant.now();
	    System.out.println("Instant: " + timestamp);
	    System.out.println("plus: " + timestamp.plus(1, ChronoUnit.HOURS));
	    System.out.println("minus: " + timestamp.minus(1, ChronoUnit.HOURS));
	    
	  //What was the time an hour agon
		Calendar cal = Calendar.getInstance();
		cal.roll(Calendar.HOUR, false);
		System.out.println("Calendar: " + cal.getTime());
	}
`